js/main.js

```diff
@@ -20,3 +20,25 @@ const lineLenght = function (line, maxLenght) {
 
 lineLenght('', 130 );
 
+const getRandomArrayElement = (elements) => elements[getRandomIntInclusive(0, elements.length - 1)];
+
+const ID = Array.from({length: 25}, (_, index) => index + 1);
+
+const URL = Array.from({length: 25}, (_, index) => `photos/${index + 1}.jpg`);
+
+const DESCRIPTION = ['Отличное фото', 'Мы с друзьями', 'Первые шаги Степы', 'Мамин юбилей', 'Вперед Спартак', 'Премьера фильма', 'Человек на Луне', 'Пикник в августе', 'Первая поездка на новом мотоцикле', 'Любовь и голуби'];
```

Comment:
Получаем массив, нужно это обозначить
```suggestion
const DESCRIPTIONS = ['Отличное фото', 'Мы с друзьями', 'Первые шаги Степы', 'Мамин юбилей', 'Вперед Спартак', 'Премьера фильма', 'Человек на Луне', 'Пикник в августе', 'Первая поездка на новом мотоцикле', 'Любовь и голуби'];
```